Sources/Networking/APIRequest.swift

```diff
@@ -100,7 +100,7 @@ public struct APIRequest {
         }
         
         if requirements.contains(.requireETagHeader), httpResponse.etag == nil {
-            throw APIRequest.Error.missingEtagInResponse
+            // throw APIRequest.Error.missingEtagInResponse
```

Comment:
TODO: Undo this change, this just makes it possible to test with a modified config from jsonblob.